bindings/rust/wasmedge-sdk/examples/run_func_in_aot_mode.rs

```diff
@@ -0,0 +1,37 @@
+use wasmedge_sdk::{
+    config::{
+        CommonConfigOptions, CompilerConfigOptions, ConfigBuilder, HostRegistrationConfigOptions,
+    },
+    params, Compiler, CompilerOutputFormat, Vm, WasmVal,
+};
+
+#[cfg_attr(test, test)]
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // create a Config context
+    let config = ConfigBuilder::new(CommonConfigOptions::new().bulk_memory_operations(true))
+        .with_compiler_config(
+            CompilerConfigOptions::new()
+                .interruptible(true)
+                .out_format(CompilerOutputFormat::Native),
+        )
+        .with_host_registration_config(HostRegistrationConfigOptions::default().wasi(true))
+        .build()?;
+
+    let wasm_file = std::path::PathBuf::from(env!("WASMEDGE_DIR"))
+        .join("bindings/rust/wasmedge-sys/tests/data/fibonacci.wasm");
+    let aot_file = std::path::PathBuf::from("fibonacci.wasm.so");
```

Comment:
You may need to add the platform check here because the `.so` extension is only available on Linux.

Reply:
Accepted. Thanks!